homepage/views.py

```diff
@@ -53,3 +54,12 @@ def add_review(request):
     else:
         form = ReviewForm()
     return render(request, 'homepage/add_review.html', {'form': form})
+
+
+def course(request, id):
+    try:
+        course = Course.objects.get(pk=id)
+        reviews = Review.objects.filter(course=id)
```

Comment:
```suggestion
        reviews = Review.objects.filter(course=id).order_by('-avg_rating')
```
WDYT?

Reply:
imo reviews should be deisplayed according to date

Reply:
IMO reviews should be displayed by the number of likes.
Meaning the review that has the most like for a specific course will be displayed first.

I think its more helpful than displaying by avg-rating or by date, since as a user i will usually want to look at the "best" reviews for the course